### PR TITLE
support UBUNTU MIRROR and modify doc

### PR DESCRIPTION
**Pull request by wen-bo-yang**
resolved https://github.com/PaddlePaddle/Paddle/issues/919
Support UBUNTU MIRROR when build Paddle docker images
Modify paddle/scripts/docker/Dockerfile
Modify paddle/scripts/docker/Dockerfile.gpu
Modify doc/getstarted/build_and_install/docker_install_en.rst

### CODE REVIEW

File: doc/getstarted/build_and_install/docker_install_en.rst [hunk 1]
```diff
@@ -142,6 +142,22 @@ to install CUDA driver and let Docker knows about it:
     export DEVICES=$(\ls /dev/nvidia* | xargs -I{} echo '--device {}:{}')
     docker run ${CUDA_SO} ${DEVICES} -it paddledev/paddle:gpu-latest
 
+    
+UBUNTU MIRROR
+-------------
+
+Building Paddle Docker image hits some wrong with apt-get update, you
```
Comment (dayhaha) [inline doc/getstarted/build_and_install/docker_install_en.rst]: Apt-get source errors may occur when building paddle docker image. For example,  the default Ubuntu source http://archive.ubuntu.com may not be accessed in China. Thus you may need to replace the default Ubuntu source with another one.         讲清楚点会不会更好些呢？

**Comment by dayhaha** [comment]
然后就是，source的问题我觉得可以在Development Using Docker这里就开始提及

在下面加上
If you are a developer from China where the default Ubuntu source http://archive.ubuntu.com can't always be accessed. We recommend you to use the following commands to build images:
docker build --build-arg UBUNTU_MIRROR="http://mirrors.163.com" -t paddle:dev -f paddle/scripts/docker/Dockerfile .

File: doc/getstarted/build_and_install/docker_install_en.rst [hunk 2]
```diff
@@ -39,12 +39,21 @@ The general development workflow with Docker and Bazel is as follows:
    code.  This image contains all the development tools and
    dependencies of PaddlePaddle.
 
-
    .. code-block:: bash
 
       cd paddle
       docker build -t paddle:dev -f paddle/scripts/docker/Dockerfile .
 
+   Apt-get source errors may occur when building paddle docker image.
```
Comment (wangkuiyi) [inline doc/getstarted/build_and_install/docker_install_en.rst]: 这里的是不是想说：

> Sometimes `docker build` might suffer from a slow network connection to the official Ubuntu apt-source servers.  In such case, we can specify an apt-source mirror server that is geologically nearer to us.  In the following example, we specified an apt-source server commonly used in China.

Comment (wangkuiyi) [inline doc/getstarted/build_and_install/docker_install_en.rst]: 我觉得不用强调中国的网络问题，在美国也有时候访问官方源很慢的。

Comment (wen-bo-yang) [inline doc/getstarted/build_and_install/docker_install_en.rst]: done

**Pull request merged**